stl/inc/atomic

```diff
@@ -131,6 +131,68 @@ _NODISCARD extern "C" bool __cdecl __std_atomic_has_cmpxchg16b() noexcept;
 #define ATOMIC_LLONG_LOCK_FREE    2
 #define ATOMIC_POINTER_LOCK_FREE  2
 
+#if _HAS_CXX20
+extern "C" {
+void __stdcall __std_atomic_wait_direct(const void* const _Storage, const void* const _Comparand,
+    const std::size_t _Size, unsigned long long& _Wait_context) noexcept;

```

Comment:
When we're at global namespace scope, we assume that `size_t` is available without qualification. This occurs below. For example, see:
https://github.com/microsoft/STL/blob/260cfafb634fe3ed64aa33242753d474170d2e2f/stl/inc/xfilesystem_abi.h#L22